web/src/compose_state.ts

```diff
@@ -253,19 +264,22 @@ export function has_full_recipient(): boolean {
         const has_topic = topic() !== "" || !realm.realm_mandatory_topics;
         return stream_id() !== undefined && has_topic;
     }
-    return private_message_recipient() !== "";
+    return private_message_recipient_ids().length > 0;
 }
 
+// TODO(evy) I think this isn't necessary anymore because `get_emails`
+// fetches users by user id and gets emails that way, so delete this
+// in a prep commit.
```

Comment:
no, it's still needed for `update_email_in_reply_to` but the `private_message_recipient_emails` isn't necessary I think?